liquibase-standard/src/main/java/liquibase/command/core/DropAllCommandStep.java

```diff
@@ -83,6 +89,10 @@ public void run(CommandResultsBuilder resultsBuilder) throws Exception {
         resultsBuilder.addResult("statusCode", 0);
     }
 
+    public SnapshotControl getSnapshotControl(CommandScope commandScope, Database database) {
```

Comment:
Could we add some javadocs here explaining why we have an empty method? Otherwise I fell someone could try to remove it in the future.